### DISCUSSION TITLE
[Feature] Share all (or some) photos with partner by default

### DISCUSSION THREAD

**Discussion by the1poet** [immich-app/immich#2320] [Feature Request]
### The feature

A useful feature from Google photos that my partner and I use is the "Partner sharing" where all photos you take are shared with them by default.
Could we enable a similar feature in Immich?
Could be implemented in many ways, such as:
- Have a list of users where all photos are shared with them by default.
- Have a default album that all photos are added to by default. Then you can choose to share said album with partner, creating a similar function.

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1668
